### DISCUSSION TITLE
Connect Container on Remote Server

### DISCUSSION THREAD

**Discussion by JayChanForlen** [docker/extension-ideas#186] [Ideas]
I have only recently been exposed to DockerDesktop and I really enjoy its monitoring platform and visual interface. I would like to know if it is possible to use DockerDesktop to connect to a server where I have already deployed Docker, so that I can use Docker Desktop to monitor my Docker on the server. Or do we currently have this type of product?

**Comment by JayChanForlen** [comment]
If u have that products?please tell me in comment. THX

**Reply by stevendouglasyagel** [reply]
XDR not THX

**Comment by ajeetraina** [comment]
@JayChanForlen Docker Desktop is not a monitoring or visual interface. It isn't designed to directly connect to and monitor a server where you've already deployed Docker. It's meant for local development on your personal machine. It provides you with a full-development environment with speed, security and choice for our developers.

Developers want to maximize their time writing code for their app — and minimize set-up and overhead — so they can ship early and often. That's where Docker Desktop provides you with set of tools like docker init, virtioFS, compose watch etc. to maximise developers producitivity.

Updated: Even though it's possible to allow Docker to accept requests from remote hosts by configuring it to listen on an IP/port as well as the Unix socket, but  it's critically important that one should understand the security implications of opening Docker to the network. One need to carefully use TLS (HTTPS) to protect the Docker daemon socket.

**Comment by benja-M-1** [comment]
Thank you @JayChanForlen for your idea! I am not aware of any extension providing this feature. Anyways, it would not really work well on DD because it is not supported yet. However, I asked, and it seems we are kinda working on something similar. I suggest you keep an eye on https://github.com/docker/roadmap/issues/47 as it is exactly what you seem looking for.
